etc/profile-m-z/telegram.profile

```diff
@@ -8,6 +8,10 @@ include globals.local
 noblacklist ${HOME}/.TelegramDesktop
 noblacklist ${HOME}/.local/share/TelegramDesktop
 
+# Allow opening hyperlinks
```

Comment:
Suggestion: use our inc file specifically designed for situations like this (instead of the two noblacklists)

include allow-bin-sh.inc

Reply:
> Suggestion: use our inc file specifically designed for situations like this (instead of the two noblacklists)
> 
> include allow-bin-sh.inc

Nice suggestion, thank you @glitsj16.